pkg/liqo-controller-manager/namespace-controller/namespace_controller.go

```diff
@@ -1,147 +1,82 @@
-/*
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-   http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
 package namespacectrl
 
 import (
 	"context"
 
-	mapsv1alpha1 "github.com/liqotech/liqo/apis/virtualKubelet/v1alpha1"
-	liqoconst "github.com/liqotech/liqo/pkg/consts"
-
 	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/klog"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	ctrlutils "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	offv1alpha1 "github.com/liqotech/liqo/apis/offloading/v1alpha1"
+	liqoconst "github.com/liqotech/liqo/pkg/consts"
 )
 
-// NamespaceReconciler changes DesiredMapping field of NamespaceMaps if a remote Namespace must be created or deleted.
+// NamespaceReconciler covers the case in which the user adds the enabling liqo label to his namespace, and the
+// NamespaceOffloading resource associated with that namespace is created, if it is not already there.
 type NamespaceReconciler struct {
 	client.Client
 	Scheme *runtime.Scheme
 }
 
 const (
-	mappingLabel                 = "mapping.liqo.io"
-	offloadingLabel              = "offloading.liqo.io"
-	offloadingPrefixLabel        = "offloading.liqo.io/"
-	namespaceControllerFinalizer = "namespace-controller.liqo.io/finalizer"
+	annotationKey   = "resource-controlled-by"
+	annotationValue = "This resource is created by the Namespace Controller"
```

Comment:
Would be more verbose in these names (for the sake of clarity):
```suggestion
	nsCtrlAnnotationKey   = "liqo.io/resource-controlled-by"
	nsCtrlAnnotationValue = "This resource is created by the Namespace Controller"
```

Reply:
it sounds clearer, thank you